### DISCUSSION TITLE
2 factor authentication expires after a few hours since MacOS Sonoma was introduced

### DISCUSSION THREAD

**Discussion by genfersee** [icloud-photos-downloader/icloud_photos_downloader#696] [Q&A]
Dear all,
Since the introduction of MacOS Sonoma, it looks like that a 2 factor authentication is required every few hours instead of several weeks as it was the case before. Did you also experience this new behavior? Is there any way to improve that?
Best regards,

**Comment by boredazfcuk** [comment]
Not here. I renewed my MFA cookie a couple of days ago and it's valid until the end of December.

**Comment by genfersee** [comment]
Ok I understood that the issue was caused by the fact that I disabled iCloud access via the web. Now back ok :) Thanks!
